js/turtles.js

```diff
@@ -974,10 +974,19 @@ Turtles.TurtlesView = class {
             document.body.appendChild(cont);
             cont.style.display = second ? "block" : "none";
             cont.setAttribute("id", "buttoncontainerTOP");
+            __makeCollapseButton();
             __makeExpandButton();
             __makeClearButton();
             __makeGridButton();
-            __makeCollapseButton();
+            const $j = jQuery.noConflict();
```

Comment:
Just two calls. Is this needed?

Reply:
Yes, earlier only the first 3 tooltip calls worked due to reasons unknown. So now, after all the methods are called, I just added this to add tooltips to each of the 4 divs. However, this could be done without declaring `$j` is not needed here. This line is left here by mistake for I was trying something else.
I am committing the deletion of this line.

Reply:
This could be done without declaring `$j` and directly using `jQuery.noConflict()`. Should I go ahead and remove `$j`?

Reply:
I removed this constant variable in the latest commit. The tooltips are working as expected. Please verify this @meganindya.